lib/modules/person.js

```diff
@@ -65,6 +49,9 @@ class PersonModule {
     }
 
     async init() {
+        // The adapter config stores "foo" whereas the real user ID is "system.users.foo".
```

Comment:
No, this comment is wrong. The username is stored. That does not necessarily have any connection to the user id.

Reply:
My adapter config stores (defaultUser) "guest" for "system.user.guest" with the name "Guest". The name is IMHO just metadata and should not be used for comparisons. system.user.guest's name might be "Peter" or any other value.